resources/trusted_certificate.rb

```diff
@@ -26,6 +26,21 @@
 property :content, String, required: [:create]
 property :sensitive, [true, false], default: true
 
+property :certificates_dir, String, default: lazy {
+  case node['platform_family']
+  when 'debian'
+    '/usr/local/share/ca-certificates'
+  when 'suse'
+    '/etc/pki/trust/anchors/'
+  else # probably RHEL
+    '/etc/pki/ca-trust/source/anchors'
+  end
+}
+
+def certificate_path
+  "#{certificates_dir}/#{certificate_name}.crt"
+end
```

Comment:
This should be moved to the `action_class` block and updated to the following:
```suggestion
def certificate_path
  "#{new_resource.certificates_dir}/#{new_resource.certificate_name}.crt"
end
```

Reply:
Although it's accessed by the action classes I _think_ this needs to stay on the outer resource class so it can be accessed externally as per the original feature request in #20.

Reply:
This cannot be inside `action_class` otherwise this won't work:

```
cert = trusted_certificate 'my_corp_remote' do
  action :create
  content 'http://www.example.com/my_corp_remote.crt'
end

cert.certificate_path
```

Where `cert.certificate_path` is used for example in a `template` which configuring the application settings.

Reply:
Would you mind adding a comment here pointing back to #20 as to why it's outside of `action_class`? That way we don't have to ask about this again down the road :)

Reply:
Done